Tests/PointFreeTests/EmailTests/__Snapshots__/InviteEmailTests/testInviteAcceptance.1.html

```diff
@@ -79,16 +79,17 @@
                            bold
                            ts-d-r2
                            ts-m-r1_5
-                           lh-2"
-                    class="m-pb2"
+                           lh-2
+                           m-pb2"
```

Comment:
this is a legit snapshot fix. in this view I was accidentally adding two `class` attributes, so while converting I merged it into a single `class`.